### PR TITLE
Add Underline and outboundlink to header. 

### PR DESCRIPTION
**Pull request by Dieterrr**
## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Made link on Courses card header outbound and added underline on header title. 

## Test instructions

This PR can be tested by following these steps:

* Yarn link yoast-components in this branch and build yoastseo. 
* Go to the courses page click on the header (image or title) and see the link opens in a new tab.
* Hover over the header and see the title gets an underline. 

Fixes #[11906](https://github.com/Yoast/wordpress-seo/issues/11906)

### CODE REVIEW

File: composites/CoursesOverview/Card.js [hunk 1]
```diff
@@ -55,10 +64,10 @@ class Card extends React.Component {
 
 		if ( this.props.header.link ) {
 			return (
-				<HeaderLink href={ this.props.header.link }>
+				<OutboundHeaderLink href={ this.props.header.link }>
```
Comment (afercia) [inline composites/CoursesOverview/Card.js]: we should use `rel={ null }` to omit noopener and noreferrer, as these links point to yoast.com

**Comment by afercia** [comment]
CR 👍 

Pushed a few changes:
- use empty alt attributes for the images, as recommended in https://www.w3.org/TR/WCAG20-TECHS/H2.html the link text is already announced, no need to repeat it in the imate alt
- added the courses overview in the standalone examples, under the "Components" tab
- RTL styles for the checkmark icon
- CSS clean up and standardize the buttons effect on :active
- use `rel={ null }` to omit the `rel="noopener noreferrer"` attributes, as these are links that point to yoast.com

To test the RTL styles:
- go in the standalone app: yarn start and go to http://localhost:3333/
- go in the "Components" tab and press the "Change language direction" button 
- verify the checkmarks position is inverted and the Cards look OK

**Comment by IreneStr** [comment]
Acceptance 🚧 
- The underlinings in safari are blue instead of pink/purple:
schermafbeelding 2019-01-02 om 12 05 42
- UX question @afercia @hedgefield: the title is also underlined when the image is hovered. Is that intentional/desirable?

**Comment by hedgefield** [comment]
I suppose that's okay as they both lead to the same place. But whether it's intentional, I don't know.

**Comment by afercia** [comment]
Last commit:
- fixes hover style in Safari
- resets the blue box-shadow _while_ clicking

**Comment by IreneStr** [comment]
Acceptance 👍 I've pushed one minor change to prevent the blue box-shadow after clicking in Chrome.

**Comment by afercia** [comment]
> I've pushed one minor change to prevent the blue box-shadow after clicking in Chrome.

Worth noting this change also removes the focus style when using the keyboard: the only indicator now is the underline on the text, which is not great but at least it's something. The persistence of the style "after clicking" is expected, as at that point the link is no more active but it does have focus.

We should try to avoid to change the focus style when the change is not an accessibility improvement 🙂

**Pull request merged**